cluster/store.go

```diff
@@ -0,0 +1,25 @@
+package cluster
+
+import (
+	"context"
+	"fmt"
+	"go.etcd.io/etcd/client"
+)
+
+const storePrefix = "store"
+
+type KVStore struct {
+	kapi client.KeysAPI
+}
+
+func NewKVStore(ctx context.Context, etcdAddr string) (*KVStore, error) {
+	cfg := client.Config{Endpoints: []string{etcdAddr}}
```

Comment:
I was thinking about if the `Config` should be created by the params of the function, or if the `Config` should be in the params, and I decided I'll do it the same way @JackyChiu did it.

1. It means we can do less validation
2. It eliminates a test case as well. Because we always set `Endpoints` properly, then `client.New(cfg)` can never return an error.

Reply:
That being said, is it worth returning an error on empty `etcdAddr` ? I don't think so...

Reply:
I'm not against having the `Config` as a param but I also didn't see a benefit to having it be a param. It also made sense to me that the KVStore might want to set some defaults in the client.Config 

> Because we always set Endpoints properly, then client.New(cfg) can never return an error.

client.New will still return an error if the addr can't be parsed

Reply:
> client.New will still return an error if the addr can't be parsed 

Oh damn, idk how the test passed on that, I guess I'm wrong

Reply:
Ya I looked at the parse function, it only returns error on empty slice I think.